janusgraph-solr/src/test/resources/solr/core-template/schema.xml

```diff
@@ -547,6 +547,7 @@
                prefixTree="geohash"/>
```

Comment:
No the client is still Solr 6, where correct package is `org.locationtech` ... it's just that the test schema.xml can be used when testing against other Solr server versions.